src/Engine/Mandarin/Mandarin.cpp

```diff
@@ -77,7 +77,7 @@ const BPMF BPMF::FromHanyuPinyin(const std::string& str) {
   // NOLINTBEGIN(bugprone-branch-clone)
 
   // the y exceptions fist
-  if (0) {
+  if (false) {  // NOLINT(readability-simplify-boolean-expr)
```

Comment:
Feel free to remove the initial `if (false)`. The code was originally written such that all branches could fit in single lines:
```
if (0) {}
else if (...) { ... }
else if (...) { ... }
```

So that everything could read like a look-up table, and the first `if (0)` was there to "remove the first true branch". But now that we apply clang-format throughout, that is moot.

Reply:
fixed in https://github.com/openvanilla/fcitx5-mcbopomofo/pull/84/commits/fe162788bb678085edc195ab09e40013eea0f95e